cmd/api/deploy.sh

```diff
@@ -14,4 +14,9 @@ go run github.com/tailscale/mkctr@latest \
   --push \
 /usr/local/bin/api
 
-flyctl deploy --detach -i registry.fly.io/al-prod:latest -a al-prod
+flyctl deploy --detach \
+	-i registry.fly.io/al-prod:latest \
+	-a al-prod \
+	-e "DD_ENV=production" \
```

Comment:
just so i'm clear, we are setting these here instead of fly.toml so that people who run their own aren't forced to run with DD?

Reply:
yeah, figure we put env specific to us in the deploy script